### PR TITLE
Increase postgres timeouts

### PR DESCRIPTION
**Pull request by sduckett**
Some states have feeds with millions of errors in their street segment
files. The queries and CSV generation take more time that we were
allowing, and when the timeout hit, we'd return them a file with only
header rows.

Three minutes seems like a lot, but it allows the queries to finish. On my dev machine, runtime is ≈ 2.5 minutes; on the staging nodes it is ≈ 1 minute.

### CODE REVIEW

File: pg/conn.js [hunk 1]
```diff
@@ -2,15 +2,35 @@ var resp = require('./response.js');
...
+  max: 10, // clients in the connection pool
+  idleTimeoutMillis: 300000
+};
+
+var pool = new pg.Pool(config);
+
+var queryFromPool = function(callback) {
+  pool.connect(function(err, client, done) {
+    if(err) {
+      return console.error('error fetching client from pool', err);
```
Comment (tie-rack) [inline pg/conn.js]: This used to be `logger.crit`. Why the change? And do messages sent to `console` end up using the logging configuration so that messages in production end up in our log aggregator? I'm guessing not.

Comment (sduckett) [inline pg/conn.js]: No, that shouldn't have changed.

Comment (tie-rack) [inline pg/conn.js]: So what about the addition of `console.error` to `logger.error` below? Do we need both, or would `logger.error` be enough?

Comment (sduckett) [inline pg/conn.js]: > Do we need both, or would `logger.error` be enough?

I'd like it to be enough, but didn't we run into some problem with that level not making it to the logs?

Comment (tie-rack) [inline pg/conn.js]: Hmm... maybe. But `console.error` is definitely not going to make it to Loggly.

**Comment by sduckett** [comment]
I've confirmed that Loggly is actually picking up `error` severity now. No more writing `logger.info("[ERROR] mess");`

**Comment by tie-rack** [comment]
Heck yes!
